<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="shipLayout"/>
   </widget>
   <widget class="QTextEdit" name="m_ptxtInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>831</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_ptxtInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>831</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>831</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="fieldLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="placeShipBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>185</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: none;
    background-image: url(images/btn1.png);
}

QPushButton:Pressed {
border:none;
background-image: url(images/btn1hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>185</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: none;
    background-image: url(images/clearbtn.png);
}

QPushButton:Pressed{
border:none;
background-image: url(images/clearbtnhover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="DoneBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>130</y>
      <width>185</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: none;
    background-image: url(images/donebtn.png);
}

QPushButton:Pressed {
border:none;
background-image: url(images/donebtnhover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
